spec/unit/TrimString.spec.js

```diff
@@ -0,0 +1,36 @@
+"use strict";
+const { trimString } = require("../../lib/util/TrimString.js");
+
+describe("trimString", function() {
+    it("should not cut unicode characters in half", (done) => {
+        const input = "lol 😅";
+        const result = trimString(input, 5);
+        expect(result).toEqual(input);
+
+        done();
```

Comment:
Shouldn't need this for sync tests. Shouldn't need it for async tests either if you return a promise :)

Reply:
Indeed, good catch. Fixed in 9cbb200e